src/graphqlTypes.ts

```diff
@@ -0,0 +1,295 @@
+export interface GQLTaxonomyEntity {
```

Comment:
Alle de typene kommer fra graphql `schema.d.ts`

Reply:
Må du definere alle disse typee?

Reply:
Egentlig så trengte jeg kun `GQLTopic` men så avhenger den av `GQLArticle`, `GQLResource`, `GQLTaxonomyEntity` og et par andre, og alle de igjen avhenger av masse annet så det balla på seg litt fort. Kunne kjøre `any` på noen av de men føltes dirty ut og det er bedre å ha god utgangspunkt for fremtidig typescript på ndla-frontend tenker jeg.